docs/src/docs/usage/faq.mdx

```diff
@@ -40,3 +40,9 @@ Long answer:
 
 Because the first run caches type information. All subsequent runs will be fast.
 Usually this options is used during development on local machine and compilation was already performed.
+
+## How does `golangci-lint` find go source files?
+
+`golangci-lint` uses go modules to find golang files. If your project is a go module, it should *just work*.
```

Comment:
> `golangci-lint` uses go modules to find golang files.

This is false or at least not precise, the use of go modules by default is related to the go version behavior.

<details>
<summary>go1.11</summary>

```console
$ go version
go version go1.11.13 linux/amd64
$ env | rg GO111MODULE
$ go env | grep GO111MODULE
$ golangci-lint run

```

</details>

<details>
<summary>go1.12</summary>

```console
$ go version
go version go1.12.17 linux/amd64
$ env | rg GO111MODULE
$ go env | grep GO111MODULE
$ golangci-lint run
```

</details>

<details>
<summary>go1.13</summary>

```console
$ go version
go version go1.13.15 linux/amd64
$ env | rg GO111MODULE
$ go env | grep GO111MODULE
GO111MODULE=""
$ golangci-lint run
```

</details>


<details>
<summary>go1.14</summary>

```console
$ go version
go version go1.14.15 linux/amd64
$ env | rg GO111MODULE
$ go env | grep GO111MODULE
GO111MODULE=""
$ golangci-lint run
```

</details>

<details>
<summary>go1.15</summary>

```console
$ go version
go version go1.15.9 linux/amd64
$ env | rg GO111MODULE
$ go env | grep GO111MODULE
GO111MODULE=""
$ golangci-lint run
```

</details>

<details>
<summary>go1.16</summary>

```console
$ go version
go version go1.16.1 linux/amd64
$ go env | grep GO111MODULE
GO111MODULE=""
$ golangci-lint run

ERRO Running error: context loading failed: no go files to analyze 
```

</details>

Reply:
Should this specifically state this applies to go 1.16 and up?

Reply:
The relation between the local go version and `GO111MODULE` exists since go1.11.

If `GO111MODULE` is `on`, even with go<1.16, `golangci-lint` will use Go modules.

Reply:
To be precise, `golangci-lint` relies on Go internal behavior, there is no `GO111MODULE` related code inside `golangci-lint` code base.